### PR TITLE
Update analytics and configs in connectedcup

### PR DESCRIPTION
**Pull request by charithag**

### CODE REVIEW

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/analytics-view.js [hunk 1]
```diff
@@ -17,17 +17,26 @@
...
     var deviceType = context.uriParams.deviceType;
     var deviceId = request.getParameter("deviceId");
 
-    if (deviceType != null && deviceType != undefined && deviceId != null && deviceId != undefined) {
+    if (devices) {
+        return {
+            "devices": stringify(devices),
+            "backendApiUri": devicemgtProps["httpsURL"] + "/android_sense/stats/"
+        };
+    } else if (deviceType != null && deviceType != undefined && deviceId != null && deviceId != undefined) {
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/analytics-view.js:29]: can simply use ->
if (deviceType && deviceId) {}

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/analytics-view.js [hunk 2]
```diff
@@ -17,17 +17,26 @@
...
         var deviceModule = require("/app/modules/device.js").deviceModule;
         var device = deviceModule.viewDevice(deviceType, deviceId);
         if (device && device.status != "error") {
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/analytics-view.js:32]: Use double quoted object key access for reading JSON attributes as it's considered to be more reliable.
Ex: person.name as person["name"]

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 3]
```diff
@@ -0,0 +1,260 @@
...
+            unstack: true,
+            stack: false,
+            xScale: d3.time.scale(),
+            padding: {top: 0.2, left: 0.02, right: 0.02, bottom: 0.2},
+            series: []
+        };
+
+        if (devices) {
+            for (var i = 0; i < devices.length; i++) {
+                graphConfig['series'].push(
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:71]: Be consistent on using quotes for string definitions. use double quotes as the standard.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 4]
```diff
@@ -0,0 +1,260 @@
...
+                        {
+                            'color': palette.color(),
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:73]: Be consistent on using quotes for string definitions. use double quotes as the standard.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 5]
```diff
@@ -0,0 +1,260 @@
...
+                            'data': [{
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:74]: Be consistent on using quotes for string definitions. use double quotes as the standard.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 6]
```diff
@@ -0,0 +1,260 @@
...
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': devices[i].name
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:78]: Be consistent on using quotes for string definitions. use double quotes as the standard.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 7]
```diff
@@ -0,0 +1,260 @@
...
+                            'name': "y"
+                        },
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': "z"
+                        }
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:107]: same as above.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 8]
```diff
@@ -0,0 +1,260 @@
...
+            } else {
+                graphConfig['series'].push(
+                        {
+                            'color': palette.color(),
+                            'data': [{
+                                x: parseInt(new Date().getTime() / 1000),
+                                y: 0
+                            }],
+                            'name': $("#connectedcup-details").data("devicename")
+                        });
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:118]: same as above.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 9]
```diff
@@ -0,0 +1,260 @@
...
+        graph.render();
+
+        var xAxis = new Rickshaw.Graph.Axis.Time({
+            graph: graph
+        });
+        xAxis.render();
+
+        var yAxis = new Rickshaw.Graph.Axis.Y({
+            graph: graph,
+            orientation: 'left',
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:132]: Be consistent on using quotes for string definitions. use double quotes as the standard.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 10]
```diff
@@ -0,0 +1,260 @@
...
+
+        var legend = new Rickshaw.Graph.Legend({
+            graph: graph,
+            element: document.getElementById("legend-" + type)
+        });
+
+        var hoverDetail = new Rickshaw.Graph.HoverDetail({
+            graph: graph,
+            formatter: function (series, x, y) {
+                var date = '<span class="date">' +
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:152]: Be consistent on using quotes for string definitions. use double quotes as the standard for outer strings and single quotes for inner strings.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js [hunk 11]
```diff
@@ -0,0 +1,260 @@
...
+                           moment((x + tzOffset) * 1000).format('Do MMM YYYY h:mm:ss a') + '</span>';
+                var swatch = '<span class="detail_swatch" style="background-color: ' +
```
Comment (DilanUA) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:154]: Be consistent on using quotes for string definitions. use double quotes as the standard for outer strings and single quotes for inner strings.

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/analytics-view.js (hunk 1 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/analytics-view.js:29]: Fixed in PR wso2#179

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/analytics-view.js (hunk 2 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/analytics-view.js:32]: How bracket notation could be more reliable? It is all about the language semantics. Using dot notation is permitted in JS as long as attribute name is a valid identifier.
http://stackoverflow.com/a/4968448/1391687

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 3 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:71]: Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR wso2#179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 4 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:73]: Single quote is the standard.
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 5 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:74]: Single quote is the standard.
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 6 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:78]: Single quote is the standard.
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 7 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:107]: Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR wso2#179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 8 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:118]: Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR wso2#179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 9 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:132]: Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR wso2#179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 10 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:152]: Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR wso2#179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

File: modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js (hunk 11 above)
Comment (charithag) [inline modules/samples/connectedcup/component/ui/src/main/resources/jaggeryapps/devicemgt/app/units/cdmf.unit.device.type.connectedcup.analytics-view/public/js/connectedcup.js:154]: Single quote is the standard. So need to change double quoted strings to single quoted. Fixed in PR wso2#179
https://google.github.io/styleguide/javascriptguide.xml?showone=Strings#Strings

**Pull request merged**